fundingmanager.go

```diff
@@ -812,7 +812,7 @@ func (f *fundingManager) advanceFundingState(channel *channeldb.OpenChannel,
 			// network.
 			// TODO(halseth): could do graph consistency check
 			// here, and re-add the edge if missing.
-			fndgLog.Debugf("ChannlPoint(%v) with chanID=%v not "+
+			fndgLog.Debugf("ChannlPoint(%v) with chan_id=%x not "+
```

Comment:
also fix `ChannlPoint`?

Reply:
lol i didn't see that, also fixed a bunch of other camelcase logs